gulpfile.js

```diff
@@ -77,37 +78,36 @@ gulp.task('build', [
 gulp.task('build-prefetcher', ['compile-lib', 'compile-client'], () => {
   return gulp
   .src('client/next-prefetcher.js')
-  .pipe(webpack({
-    quiet: true,
+  .pipe(webpackStream({
     output: { filename: 'next-prefetcher-bundle.js' },
     plugins: [
-      new webpack.webpack.DefinePlugin({
+      new webpack.DefinePlugin({
         'process.env': {
           NODE_ENV: JSON.stringify('production')
         }
       })
     ],
     module: {
-      loaders: [
+      rules: [
         {
           test: /\.js$/,
           exclude: /node_modules/,
-          loader: 'babel',
-          query: {
-            'babelrc': false,
-            'presets': [
+          loader: 'babel-loader',
+          options: {
```

Comment:
here we could decorate this with `options.babel()` to make it easier to config babel for next?